Packs/Zscaler/Integrations/Zscaler/Zscaler.py

```diff
@@ -732,13 +735,43 @@ def sandbox_report(md5, details):
     return response
 
 
+def login_command():
+    auth = login()
+    jsession_id = auth[:auth.index(';')]
+    DEFAULT_HEADERS['cookie'] = jsession_id
+    readable_output = '### Created Zscaler Session\nSession ID: {}'.format(jsession_id)
+    return CommandResults(
+        outputs_prefix='Zscaler.SessionID',
+        outputs=jsession_id,
+        readable_output=readable_output,
+        raw_response=auth)
+
+
+def logout_command():
+    logout()
+    return "API session logged out of Zscaler successfully."
+
+
+def activate_command():
+    activate_changes()
+    return "Changes have been activated successfully."
```

Comment:
i know it is py2 and no type assertion, but please consider still returning the commandResults object for consistency.

Reply:
Even if there was a type assertion, it wouldn't catch this, since `return_results` accepts `str` as valid input. I feel it's more cumbersome, but I'll switch to it 👍